src/include/btree_cmp_inline.h

```diff
@@ -166,6 +300,143 @@ __wt_compare_bounds(WT_SESSION_IMPL *session, WT_CURSOR *cursor, WT_ITEM *key, u
     return (0);
 }
 
+#ifdef HAVE_X86INTRIN_H
+/*
+ * __lex_compare_skip_gt_16 --
+ *     Lexicographic comparison routine for data greater than 16 bytes, skipping leading bytes.
+ *     Returns: < 0 if user_item is lexicographically < tree_item = 0 if user_item is
+ *     lexicographically = tree_item > 0 if user_item is lexicographically > tree_item We use the
+ *     names "user" and "tree" so it's clear in the btree code which the application is looking at
+ *     when we call its comparison function. Some platforms like ARM offer dedicated instructions
```

Comment:
The last sentence is duplicated. Plus I'm not sure it makes sense to talk about arm on the x86 impl

Reply:
fixed